tests/unittests/OperatorTest.cpp

```diff
@@ -44,6 +44,8 @@ class InterpAndCPU : public Operator {};
 
 class InterpOnly : public Operator {};
 
+class OpenCLOnly : public Operator {};
```

Comment:
not sure if we want this specific one, seems it's better to run quantization ops tests on all backends.